src/numerics/include/metaphysicl/dualnumber.h

```diff
@@ -528,6 +643,25 @@ DualNumber_std_binary(min,
   (a.value() > b.value()) ? b.derivatives() : a.derivatives())
 DualNumber_std_binary(fmod, a.derivatives())
 
+#if __cplusplus >= 201103L
+DualNumber_equivfl_binary(pow)
+DualNumber_equivfl_binary(fmod)
+DualNumber_std_binary(remainder, a.derivatives())
+DualNumber_equivfl_binary(remainder)
+DualNumber_equiv_binary(fmax, max)
+DualNumber_equivfl_binary(fmax)
+DualNumber_equiv_binary(fmin, min)
+DualNumber_equivfl_binary(fmin)
+DualNumber_std_binary(fdim, if_else(a.value() > b.value(),
+                                    a.derivatives() - b.derivatives(), 0))
+DualNumber_equivfl_binary(fdim)
+DualNumber_std_binary(hypot, (a.value()*a.derivatives() +
```

Comment:
I have declarations and definitions for the ternary form of this in my branch that could be put in this PR or in a follow-on

Reply:
See https://github.com/lindsayad/MetaPhysicL/commit/e27b37dd6b3cf45d7168f840cfc4b5c8ed9c55f2 and https://github.com/lindsayad/MetaPhysicL/commit/11f220a1b1817ec6064c31884d345dde7dadcce5

Reply:
That's excellent; thanks!